velodyne_convert_node:
    ros__parameters:
        calibration: VLP16db.yaml
        model: VLP16
        target_frame: velodyne
        fixed_frame: velodyne
        min_range: 0.9
        max_range: 130.0
        view_direction: 0.0
        organize_cloud: true
